src/utils.ts

```diff
@@ -102,3 +102,19 @@ export async function cleanOutDir(
     }
     await fs.mkdir(outDirPath, { recursive: true })
 }
+
+export function makePortablePath(path: string): string {
+    // First normalize and convert all backslashes to forward slashes
+    let cleaned = normalize(path).replace(/\\/g, '/')
+
+    // Remove Windows drive letter prefix
+    cleaned = cleaned.replace(/^[a-zA-Z]:\//, '')
+
+    // Remove any leading slashes
+    cleaned = cleaned.replace(/^\/+/, '')
+
+    // Remove any duplicate slashes
+    cleaned = cleaned.replace(/\/+/g, '/')
+
+    return cleaned
+}
```

Comment:
Do you want to convert Windows paths to POSIX?

It is better to use `path.posix.*` to convert paths, it is easier and more reliable way